core/domain/platform_parameter_registry.py

```diff
@@ -541,3 +541,59 @@ def _to_platform_parameter_model(
     # TODO(release-scripts#130): Update post the first trial release.
     default=feconf.NOREPLY_EMAIL_ADDRESS
 )
+
+Registry.create_platform_parameter(
+    ParamName.MAILCHIMP_AUDIENCE_ID,
+    'Audience ID of the mailing list for Oppia in Mailchimp.',
+    platform_parameter_domain.DataTypes.STRING,
+    # TODO(release-scripts#134): Update post the first trial release.
```

Comment:
Just to clarify, what are we planning to update each of these defaults to? I don't see this information stated in the linked issue.

Reply:
Hmm, that would be derived from the existing feconf/third party constant updates we have per server [here](https://github.com/oppia/release-scripts/tree/master/deploy_data). Is that what you were asking or did I misunderstand something here?

Reply:
Hi @ankita240796 -- yeah, sorry, I think I might be confused. Are you saying that you're planning to take those (server-specific) constants and hardcode them within this (public) file, which has the general config that would be deployed to all 3 servers? If so then I don't think I understand what the plan is exactly (and also why we need to update these lines of code after this PR rather than in this PR).

Please could you clarify? (It might also be worth updating the checklist in the TODO-linked issue with the exact steps that need to be taken -- I'm not sure whether this TODO is reflected there or not.)

Thanks!

Reply:
Also, just noting that the "default" value here is the default value that gets used for the platform param if a new value hasn't yet been specified by the admin on the specific server. (Adding @lkbhitesh07 to double-confirm this.)

Reply:
- The correct (non-default) values of these platform parameters will be set using the server specific constants present in release repo by the RC during the first release post testing is done.
- The default will be same as what the defaults are currently in feconf. For instance, if default in feconf for `MAILCHIMP_AUDIENCE_ID` is some dummy value, we just copy it over here too. The reason for the comment is that we can directly put that dummy here and remove the mention from feconf altogether. This will also happen only after testing since till then feconf values are still being updated during releases and being used across the codebase. Is there any issue in doing this?

Reply:
Oh I see. So the default value here would match the dummy value in the current codebase, right?  If so, I would suggest just putting the dummy value here directly in this PR then, and dropping the TODO. It won't have any effect until you start using the params.

Reply:
Done.

Reply:
> Also, just noting that the "default" value here is the default value that gets used for the platform param if a new value hasn't yet been specified by the admin on the specific server. (Adding @lkbhitesh07 to double-confirm this.)

That is correct @seanlip